examples/rtic-tick.rs

```diff
@@ -20,8 +23,10 @@ mod app {
         led: PC13<Output>,
     }
 
-    #[monotonic(binds = TIM2, default = true)]
-    type MicrosecMono = MonoTimerUs<pac::TIM2>;
+    //#[monotonic(binds = TIM2, default = true)]
+    //type MicrosecMono = MonoTimerUs<pac::TIM2>;
```

Comment:
Seems duplicate?

Reply:
no, it is not

Reply:
I mean, what is the purpose of having a commented block of 2 lines that is identical to the 2 lines before?

Reply:
Those "2 lines before" are deleted lines. Red.